src/gen_helper.ts

```diff
@@ -0,0 +1,157 @@
+import * as graphene from "graphene-pk11";
+import {TEST_KEY_ID, TEST_KEY_LABEL} from "./const";
+
+
+function gen_AES(session: graphene.Session, name: string, len: number, token = false, test:boolean) {
+    return session.generateKey(
+        graphene.KeyGenMechanism.AES,
+        {
+            keyType: graphene.KeyType.AES,
+            id: TEST_KEY_ID,
+            label: test?TEST_KEY_LABEL:name+'-Key',
```

Comment:
The idea is this: Since `name` was oddly set and never used the previous version of `gen_helper`, we can change how it's used, as you have here. I think setting `label: name` should be fine.